### ISSUE TITLE
Use sbt-js-engine instead of Rhino directly

### ISSUE THREAD

**Issue by mdedetrich** [joescii/sbt-jasmine-plugin#13]
A lot of javascript related plugins in Scala land (particularly the ones from Play) use sbt-js-engine as a backend for running Javascript, rather than using Rhino (or any other engine directly). Using sbt-js-engine does provide various benefits

- It has its own backend for pluggable engines
- Better supported (this is used by Play for their plugins, such as https://github.com/sbt/sbt-coffeescript or https://github.com/sbt/sbt-rjs)
- Automatically supports a new engine should it get added (i.e. you get Nashorn support when it gets added to sbt-js-engine)
- Able to run tasks in parallel on multiple cores for better performance

Should we consider using sbt-js-engine instead of rhino for sbt-jasmine-plugin?

**Comment by joescii** [comment]
Yeah, that would definitely be a nice enhancement.  I think I looked over it about a year ago, but didn't find time to implement the changes.

**Comment by laguiz** [comment]
+1 for this one
